fast_api_als/routers/three_pl.py

```diff
@@ -29,15 +37,22 @@ async def reset_authkey(request: Request, token: str = Depends(get_token)):
 
 @router.post("/view_authkey")
 async def view_authkey(request: Request, token: str = Depends(get_token)):
-    body = await request.body()
-    body = json.loads(body)
+    try:
+        body = await request.body()
+        body = json.loads(body)
+        logger.info('Body of request parsed successfully')
+    except Exception as e:
+        logger.error(f'Parsing of body failed due to {e}')
```

Comment:
what is the specific type of exception that is caught here and the similar instances in the same file?